wildfly-standalone-centos8/README.md

```diff
@@ -48,16 +46,15 @@ Following is the Architecture:
 
 ![alt text](images/wildfly-arch.png)
 
-To learn more about the WildFly 18.0.0.Final, visit:
-https://docs.wildfly.org/18/
+To learn more visit the [Wildfly Documentation](https://docs.wildfly.org/18/) site.
 
 ## Licenses, Subscriptions and Costs 
 
-This template uses an Azure CentOS 8 image which is a Pay-as-you-go (PAYG) VM image and does not require the user to license. The VM will be licensed automatically after the instance is launched for the first time and user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/#linux) for pricing details. WildFly is free to download and use and does not require a Red Hat Subscription or any License, for more details click [here](https://www.wildfly.org/).
+This template uses an Azure CentOS 8 image which is a Pay-As-You-Go (PAYG) VM image and does not require the user to license. The VM will be licensed automatically after the instance is launched for the first time and the user will be charged hourly in addition to Microsoft's Linux VM rates.  [Linux Virtual Machine](https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/#linux) provides pricing details. WildFly is free to download and use and does not require a Red Hat Subscription or License.
```

Comment:
Change accommodated – Heading changed to “Subscriptions and Costs” and paragraph under this section